lib/repo/mapping.go

```diff
@@ -82,6 +85,17 @@ func (d *deviceCollection) findDevice(device string) (*DeviceMapping, error) {
 	return nil, errors.New(missingRepo)
 }
 
+func DownloadDevicesRepository() {
```

Comment:
__[golint]__
* [golint] exported function DownloadDevicesRepository should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->